Lesson9/anna/Lesson9-aysel/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  display: grid;
+  margin: 100px auto;
+  grid-gap: 10px 10px;
+  grid-template-columns: 2fr 4fr 11fr 4fr 2fr;
+}
+img {
+  max-width: 100%;
+}
+
+.post > * {
+  grid-column: 3 / span 1;
+}
+.clock,
+.youtube-pic {
+  width: 100%;
+  grid-column: 2 / span 3;
+}
+
+h1 {
+  font-size: 42px;
+  font-weight: 700;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 52px;
+  letter-spacing: -0.011em;
+}
+
+.first-pic {
+  border-radius: 50px;
+  position: relative;
+}
+.second-pic {
+  position: relative;
+  margin-left: -14px; /*i couldnt find any other way to put this pic on the first one*/
```

Comment:
In this case, it looks alright. I'd paces this two images in a div. Then use position relative on the div and position absolute for image elements to adjust where they should be.

Reply:
I this folder, I added my version of the solution for this assignment. Feel free to look into it as some parts of the code may be helpful but still try to solve it on your own.